unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/RPCFriendsApiBridge.cs

```diff
@@ -3,13 +3,22 @@
 using DCL.Social.Friends;
 using Decentraland.Renderer.KernelServices;
 using Decentraland.Renderer.RendererServices;
+using Decentraland.Renderer.Common;
 using JetBrains.Annotations;
 using RPC;
 using rpc_csharp;
 using System;
 using System.Linq;
 using System.Threading;
 using GetFriendRequestsPayload = Decentraland.Renderer.KernelServices.GetFriendRequestsPayload;
+using KernelRejectFriendRequestReply = Decentraland.Renderer.KernelServices.RejectFriendRequestReply;
+using KernelRejectFriendRequestPayload = Decentraland.Renderer.KernelServices.RejectFriendRequestPayload;
+using KernelCancelFriendRequestReply = Decentraland.Renderer.KernelServices.CancelFriendRequestReply;
+using KernelCancelFriendRequestPayload = Decentraland.Renderer.KernelServices.CancelFriendRequestPayload;
+using RendererRejectFriendRequestReply = Decentraland.Renderer.RendererServices.RejectFriendRequestReply;
+using RendererRejectFriendRequestPayload = Decentraland.Renderer.RendererServices.RejectFriendRequestPayload;
+using RendererCancelFriendRequestReply = Decentraland.Renderer.RendererServices.CancelFriendRequestReply;
+using RendererCancelFriendRequestPayload = Decentraland.Renderer.RendererServices.CancelFriendRequestPayload;
```

Comment:
Could we simply do?
```suggestion
using Decentraland.Renderer.KernelServices;
```

Reply:
i cant, i need to import it with a custom name because classes have the same name in both services/namespaces